accounts/serializers.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework import serializers
+from accounts.models import User
+from datetime import timedelta
+from django.utils import timezone
+class UserRegistrationSerializer(serializers.ModelSerializer):
+    password2 = serializers.CharField(style={'input_type':'password'}, write_only=True)
+    class Meta:
+        # For confirm password field in Registration Request
+        model = User
+        fields = ['email','name','password','password2']
+        extra_kwargs ={
+            'password': {'write_only':True}
+        }
+    # password and confirm password2 validation
+    def validate(self, attrs):
+        password = attrs.get('password')
+        password2 = attrs.get('password2')
+        if password != password2:
+            raise serializers.ValidationError("Passwords do not match")
+        return attrs
+
+    def create(self, validated_data):
+        return User.objects.create_user(**validated_data)
+    
+class VerifyEmailSerializer(serializers.Serializer):
+    otp = serializers.CharField(max_length=6)
+    email = serializers.EmailField(max_length=255)
+    class Meta:
+        fields = ['otp', 'email']
+    
+    def validate(self, attrs):
```

Comment:
i think attrs can be renamed as request or something

Reply:
Renamed to data for better convention as the data can be request or response or a context.